service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -116,56 +116,6 @@ public String getRequestUserEmail(AuthenticatedUserRequest userRequest)
     }
   }
 
```

Comment:
Do you want to define an interface for SamService to implement? Seems like that would help with the mocking.

Reply:
Not at this time. I am trying to keep existing WSM tests working as we add the PoC stuff. My plan is to have logic like the following to SamService:
```
if (azureState.isEnabled() && userRequest.getAuthType == AuthType.BASIC) {
  ...do the mock stuff...
} else {
  ...do the real stuff...
}
```
It'll do for a PoC.